### ISSUE TITLE
Scrolling tabs using mouse wheel doesn't work when Reverse Tab Order selected and in vertical mode

### ISSUE THREAD

**Issue by kkot** [bwinton/TabCenter#988]
I move mouse over one of the tabs (not pined) and scroll mouse wheel and tabs don't scroll. It only happen when Reverse Tab Order is selected and I'm in vertical mode.

I'm using version 1.31.0. I tested it on clean profile without other extensions. 

BTW. I'm using Reverse Tab Order mainly because pinned tabs are not really pinned so I would need to scroll up a lot otherwise.

**Comment by SoftVision-CiprianMuresan** [comment]
This is most likely a dupe of #776 and is no longer reproducible on the latest master.
@ericawright, will this be included in the next release?

**Comment by kkot** [comment]
On laptop at work scrolling works. At home it doesn't I'm not sure what is different.

**Comment by kkot** [comment]
No after restart it also doesn't work at work. Probably just updated from 1.29 to 1.31 so it still worked because something was not updated.

**Comment by SoftVision-CiprianMuresan** [comment]
Ignore my previous comments.

The issue is reproducible on the latest Firefox release (51.0.1), but it is not reproducible on the latest Nightly (54.0a1). 
I used mozregression to find out what fixed the issue and it seems like the work that was done in [Bug 1316505](https://bugzilla.mozilla.org/show_bug.cgi?id=1316505) is the culprit.

@ericawright, any thoughts on this?

**Comment by ericawright** [comment]
Doesn't work in release...that's troublesome. I'll see what I can do - and thanks so much for finding the culprit!
